src/app/result/[postId]/_components/ProfileBlock.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+
+interface Props {
+  name: string;
+  imageUrl: string;
+}
+
+const ProfileBlock = ({ name, imageUrl }: Props) => {
+  return (
+    <div className='flex w-full items-center justify-start gap-4 p-4'>
+      <Image
+        src={imageUrl}
+        alt={`${name}님의 프로필 이미지`}
+        width={40}
+        height={40}
+        className='rounded-full object-cover'
+      />
+      <p className=' self-center text-sm font-bold'>{name}</p>
```

Comment:
```suggestion
      <p className='text-sm font-bold'>{name}</p>
```
다음처럼 수정하면 좋을 것 같아요 

그리고 p태그를 사용한 이유가 있을까요 ??